### DISCUSSION TITLE
Real APP_ENV ignored in bin/console; tries to load .env file

### DISCUSSION THREAD

**Discussion by aprat84** [symfony/symfony#47590] [Q&A]
I'm deploying my app in ElasticBeanstalk, and part of the deploy process is executing some commands (ie: `cache:warmup`).

The staging/production machine already has real environment variables (`APP_ENV`, `APP_DEBUG`, ...), and if I dump `$_SERVER` or `$_ENV` inside the `bin/console` script they exist, but the `bin/console` script ignores them, and tries to load the `.env` file, which obviously doesn't exist.

What am I missing here?

I'm on symfony 6.1.

**Comment by athos99** [comment]
Hello
Try the console option   
-e, --env=ENV         The Environment name. [default: "dev"]
The console ignore APP_ENV 


For buid for env et prod 
php bin/console cache:warmup --env=dev 
php bin/console cache:warmup --env=prod

**Comment by aprat84** [comment]
The `APP_ENV` variable is already set. Why the need of an extra command argument?

I have solved this by creating an empty `.env` file at deployment time, so it doesn't crash.

But still, I don't see the need of a `.env` file in production, where environment variables just exist 🤷‍♂️
